kubernetes-deployment/massdriver.yaml

```diff
@@ -81,13 +118,34 @@ params:
         enabled:
           title: Enable Public Internet Access
           type: boolean
-        host:
-          title: Hostname
-          description: TBD, is this the right field name?
-          type: string
-        path:
-          type: string
-          default: '/'
+      dependencies:
+        enabled:
+          oneOf: 
+          - properties:
+              enabled:
+                const: false
+          - properties:
+              enabled:
+                const: true
+              host:
```

Comment:
It'd be great to have this eventually be a widget based off of the connected kubernetes cluster. They specificy the subdomain and the http path and then the domain itself is a dropdown from the domains the kubernetes cluster manages.